Wake On Lan/Modules/CoreDataService/Sources/Migrations/CoreDataAppToSharedGroupMigration.swift

```diff
@@ -93,13 +93,14 @@ extension CoreDataAppToSharedGroupMigration {
                 try? fileManager.removeItem(at: url)
             }
         )
-        if let error {
+        if let error = error {
```

Comment:
Update your Xcode : )
```suggestion
        if let error {
```

Reply:
I have already got the new Xcode version, actually I made this changes when only the Xcode 13 was installed on my mac.